exporter/splunkhecexporter/client_test.go

```diff
@@ -236,6 +236,8 @@ func runMetricsExport(cfg *Config, metrics pmetric.Metrics, expectedBatchesNum i
 	assert.NoError(t, err)
 	assert.NoError(t, exporter.Start(context.Background(), componenttest.NewNopHost()))
 	defer func() {
+		// Read from channel so that the goroutine running the send operation can exit successfully
+		_ = <-capture.receivedRequest
```

Comment:
Why do we have the channel if it's not being used? Can we change the mock handler to not send the response if the channel is `nil`?

Reply:
I converted the channel to a `*chan`, so now we can check if it's nil and only block on sending if it's set.

Reply:
`chan` can be nil as well, no need to have another pointer :)

Reply:
Good to know, thanks! _Done_